### PR TITLE
Add EBNF as an appendix

### PR DESCRIPTION
**Pull request by david-a-wheeler**
Signed-off-by: David A. Wheeler <dwheeler@dwheeler.com>

### CODE REVIEW

File: metamath.tex [hunk 1]
```diff
@@ -14764,6 +14765,149 @@ \chapter{The MIU System}
...
+  _WHITECHAR+ '\$)' _WHITECHAR
+
+/* Whitespace character: (' ' | '\t' | '\r' | '\n' | '\f') */
+_WHITECHAR ::= [#x20#x09#x0d#x0a#x0c]
+\end{verbatim}
+
+This EBNF was developed as a collaboration between
+David A. Wheeler\index{Wheeler, David A.},
+Mario Carneiro\index{Carneiro, Mario}, and
+Benoit Jubin\index{Jubin, Benoit}, inspired by a request
```
Comment (benjub) [inline metamath.tex]: Thanks for your work.
I would remove ", inspired by a request (and a lot of initial work) by Benoit Jubin".  Actually, I would not mind if that paragraph was removed altogether: DAW and MC already appear as overall contributors of the book, and my contribution is in contrast negligible.

Comment (david-a-wheeler) [reply metamath.tex]: I'm okay with removing it, though I'm obviously okay with giving you credit too :-).  I'll remove the whole paragraph.

Comment (david-a-wheeler) [reply metamath.tex]: Actually, better idea: I'll turn it into a comment.  It won't show in the printed work, but it'll still be documented. Your contribution was not negligible, even if you think overwise :-).

**Comment by david-a-wheeler** [comment]
@nmegill - ok to merge?

This new EBNF appendix is a bigger change, but it was posted earlier on the mailing list for review & is the result of work between myself, Mario, and Benoit.  That should reduce the risk of any serious error.

In addition, it clearly states that in case of a conflict, section 4.1 governs.  So while I hope there's no mistake, I think that mitigates the impact of any mistake.

**Comment by nmegill** [comment]
ok to merge.

**Pull request merged**